docs/messaging/kafka-integration.md

```diff
@@ -334,14 +334,15 @@ The .NET Aspire Apache Kafka integration dos not emit distributed traces.
 
 The .NET Aspire Apache Kafka integration emits the following metrics using OpenTelemetry:
 
-- `messaging.kafka.network.tx`
-- `messaging.kafka.network.transmitted`
-- `messaging.kafka.network.rx`
-- `messaging.kafka.network.received`
-- `messaging.publish.messages`
-- `messaging.kafka.message.transmitted`
-- `messaging.receive.messages`
-- `messaging.kafka.message.received`
+- Aspire.Confluent.Kafka
```

Comment:
```suggestion
- `Aspire.Confluent.Kafka`
```

Reply:
I didn't include code blocks around the meter name to match https://learn.microsoft.com/en-us/dotnet/aspire/database/sql-server-integration?tabs=dotnet-cli%2Cssms#metrics - may want to update sql client for consistency.